src/moin/storage/middleware/indexing.py

```diff
@@ -757,18 +808,21 @@ def update(self, tmp=False):
             backend_latest_backends_revids = set(self._find_latest_backends_revids(index_all))
         finally:
             index_all.close()
-        index_latest = storage.open_index(LATEST_REVS)
-        try:
-            # now update LATEST_REVS index:
-            with index_latest.searcher() as searcher:
-                ix_revids = {doc[REVID] for doc in searcher.all_stored_fields()}
-            backend_latest_revids = {revid for name, revid in backend_latest_backends_revids}
-            upd_revids = backend_latest_revids - ix_revids
-            upd_revids = [(revids_backends[revid], revid) for revid in upd_revids]
-            self._modify_index(index_latest, self.schemas[LATEST_REVS], upd_revids, "update")
-            self._modify_index(index_latest, self.schemas[LATEST_REVS], del_revids, "delete")
-        finally:
-            index_latest.close()
+
+        # update LATEST_REVS and LATEST_IDX
```

Comment:
Maybe rather tell here WHY we have the smaller index.

Reply:
The explanation is now in lines 449-450:

```
    # Small index for the latest revisions, used for queries such as has_item, authorization checks and
    # the +index route. This index has no content or *NGRAMS, which improves query speed for large wikis
```